src/view/com/pager/PagerWithHeader.tsx

```diff
@@ -277,7 +279,7 @@ let PagerTabBar = ({
   return (
     <Animated.View
       pointerEvents={isIOS ? 'auto' : 'box-none'}
-      style={[styles.tabBarMobile, headerTransform]}>
+      style={[styles.tabBarMobile, headerTransform, t.atoms.bg]}>
```

Comment:
i was noticing scrollview content sometimes appearing under the header, this makes the header always a solid color